### ISSUE TITLE
webview crached when javascript touches it if it is not be displayd

### ISSUE THREAD

**Issue by mizchi** [atom/atom-shell#989]
MacOSX10.9: AtomShell v0.20.4

code

```javascript
document.body.innerHTML = '<webview src="https://github.com"></webview>'
var webview = document.querySelector('webiview');
console.log(webview.getUrl()); // it works.

setInterval(function(){
  console.log(webview.getUrl()); // this causes crashing at display: none
}, 1000);
document.body.style.display = 'none';
```

**Comment by neekychan** [comment]
The same problem

**Comment by paulcbetts** [comment]
This is #764 afaik

**Comment by zcbenz** [comment]
Yeah it seems like a duplicate of #764, I would like to look into this after Chrome40 gets released.

**Comment by mizchi** [comment]
@zcbenz @paulcbetts Thank you I got it

**Comment by zcbenz** [comment]
I can confirm this has been fixed in v0.21.0.
